osm-georeferencing/main.py

```diff
@@ -11,11 +10,11 @@
 import shared
 
 logging.basicConfig(format='%(asctime)s - %(message)s', level=logging.INFO)
-OA_BODY = "oa:hasBody"
-DWC_LOCALITY = "dwc:locality"
-OA_VALUE = "oa:value"
-LOCATION_PATH = "$.ods:hasEvent[*].ods:Location."
-USER_AGENT = "Distributed System of Scientific Collections"
+OA_BODY = 'oa:hasBody'
+DWC_LOCALITY = 'dwc:locality'
+OA_VALUE = 'oa:value'
```

Comment:
Nice! How did this not fail earlier? Because it were strings without quotes

Reply:
no it was double quotes i changed to single quotes to be more consistent